src/util.h

```diff
@@ -93,7 +93,7 @@ bool error(const char* fmt, const Args&... args)
 
 void PrintExceptionContinue(const std::exception *pex, const char* pszThread);
 void ParseParameters(int argc, const char*const argv[]);
-void FileCommit(FILE *fileout);
+void FileCommit(FILE *file);
```

Comment:
Besides immamentizing the heat death of the universe, the only reason for having argument names in the interface specification at all is for documentation, I don't think `file` is any clearer than `fileout`, and I don't think this change here should have been made.

Then again as @MarcoFalke already did the binary comparison it's too late to make changes, and I'll just go ahead and merge...